frontend/src/components/dashboard/customHomePage/MineSubscriptionTable.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from "react";
+import PropTypes, { objectOf, string } from "prop-types";
+import { Link } from "react-router-dom";
+import { Table, Popconfirm } from "antd";
+import { uniqBy } from "lodash";
+import * as router from "@/constants/routes";
+import * as Strings from "@/constants/strings";
+import NullScreen from "@/components/common/NullScreen";
+import CustomPropTypes from "@/customPropTypes";
+import { UNSUBSCRIBE } from "@/constants/assets";
+
+/**
+ * @class MineSubscriptionTable is a user specific table of mines they have subscribed to with the ability to unsubscribe
+ *
+ */
+
+const propTypes = {
+  handleUnSubscribe: PropTypes.func.isRequired,
+  subscribedMines: PropTypes.arrayOf(CustomPropTypes.mine).isRequired,
+  mineRegionHash: objectOf(string).isRequired,
+  mineTenureHash: objectOf(string).isRequired,
+  mineCommodityOptionsHash: objectOf(string).isRequired,
+};
+
+export class MineSubscriptionTable extends Component {
+  transformRowData = (mines, mineRegionHash, mineTenureHash, mineCommodityHash) =>
+    mines.map((mine) => ({
+      key: mine.mine_guid,
+      emptyField: Strings.EMPTY_FIELD,
+      mineName: mine.mine_name || Strings.EMPTY_FIELD,
+      mineNo: mine.mine_no || Strings.EMPTY_FIELD,
+      operationalStatus: mine.mine_status.length
+        ? mine.mine_status[0].status_labels[0]
+        : Strings.EMPTY_FIELD,
+      permit: mine.mine_permit.length ? mine.mine_permit : null,
+      region: mine.region_code ? mineRegionHash[mine.region_code] : Strings.EMPTY_FIELD,
+      commodity: mine.mine_type.length ? mine.mine_type : null,
+      commodityHash: mineCommodityHash,
+      tenure: mine.mine_type.length ? mine.mine_type : null,
+      tenureHash: mineTenureHash,
+      tsf: mine.mine_tailings_storage_facility
+        ? mine.mine_tailings_storage_facility.length
+        : Strings.EMPTY_FIELD,
+    }));
+
+  render() {
+    const columns = [
```

Comment:
At a glance this looks very close to the table definition on the mine lookup. I'm wondering if they could be shared, and the extra column for subscribe/unsubscribe appended to the columns. You might want a subscribe column on the Mine Lookup anyways. Not a request, just a thought.

Reply:
They are very similar, the main difference being `MineList` expects the array of mineIds and then maps through that to access the mine via `mine[id]`

In this situation, I am passing in an array of subscribedMines and mapping through that. 
Super trivial, but requires a bit of a refactor. 
I'll chat with Aaron/ designers to see if they want the ability to subscribe/unsub on the main mine list, and refactor at that time. 

I think that would be a good enhancement!